Tests/TestUtilities/TestUtils.cs

```diff
@@ -183,14 +172,31 @@ public static string WriteBatchFileForTest(TestContext context, string content)
     /// Creates a project info under the specified analysis root directory
     /// together with the supporting project and content files, along with GUID and additional properties (if specified)
     /// </summary>
-    public static string CreateProjectWithFiles(TestContext testContext, string projectName, string projectLanguage, string analysisRootPath, Guid projectGuid, bool createContentFiles = true, AnalysisProperties additionalProperties = null)
+    public static string CreateProjectWithFiles(
+        TestContext testContext,
```

Comment:
For method declarations we align the parameters differently, first after the parenthesis and the next ones one under the first one.